src/SLCore.Listeners.UnitTests/Logging/LoggerListenerTests.cs

```diff
@@ -40,12 +40,12 @@ public void Mef_CheckIsSingleton()
         }
 
         [TestMethod]
-        [DataRow(LogLevel.ERROR, true)]
-        [DataRow(LogLevel.WARN, true)]
-        [DataRow(LogLevel.INFO, false)]
-        [DataRow(LogLevel.TRACE, false)]
-        [DataRow(LogLevel.DEBUG, false)]
-        public void Log_LogsOnlyErrorAndWarning(LogLevel logLevel, bool logs)
+        //[DataRow(LogLevel.ERROR, false)]
+        //[DataRow(LogLevel.WARN, false)]
+        //[DataRow(LogLevel.INFO, true)]
+        //[DataRow(LogLevel.TRACE, true)]
```

Comment:
This probably shouldn't be left commented out.

Reply:
😅